cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_controller_test.go

```diff
@@ -738,19 +738,27 @@ func TestControllerNodeGroups(t *testing.T) {
 
 func TestControllerNodeGroupsNodeCount(t *testing.T) {
 	type testCase struct {
-		nodeGroups    int
-		nodesPerGroup int
+		nodeGroups            int
+		nodesPerGroup         int
+		expectedNodeGroups    int
+		expectedNodesPerGroup int
 	}
 
 	var testCases = []testCase{{
-		nodeGroups:    0,
-		nodesPerGroup: 0,
+		nodeGroups:            0,
+		nodesPerGroup:         0,
+		expectedNodeGroups:    0,
+		expectedNodesPerGroup: 0,
 	}, {
-		nodeGroups:    1,
-		nodesPerGroup: 0,
+		nodeGroups:            1,
+		nodesPerGroup:         0,
```

Comment:
Should this be called replicaCount because this what it actually is and gets interpreted as nodesPerGroups? This will increase readability of the logic.

Reply:
I agree. I have a follow-up PR that will change this for all tests as we pass in both `nodeCount` and `replicaCount`. Looking at this yesterday I don't believe we always need the distinction and replicaCount would suffice. To make this PR straightforward I would rather do that change en masse.

Reply:
> I agree. I have a follow-up PR that will change this for all tests as we pass in both `nodeCount` and `replicaCount`. Looking at this yesterday I don't believe we always need the distinction and replicaCount would suffice. To make this PR straightforward I would rather do that change en masse.

https://github.com/openshift/kubernetes-autoscaler/pull/106